### PR TITLE
new lab on instance methods

### PR DESCRIPTION
**Pull request by linda-seiter**

### CODE REVIEW

**Review by developerjaym — APPROVED** [review]

File: src/test/java/TVTest.java [hunk 1]
```diff
@@ -0,0 +1,191 @@
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class TVTest {
+
+    @Test
+    void toggleOnOff() {
+        TV tv = new TV();  //tv not on
```
Comment (kcleland0818) [inline src/test/java/TVTest.java]: Instantiate the TV in a setUp() method prior?

**Review by kcleland0818 — APPROVED** [review]

File: src/test/java/TVTest.java (hunk 1 above)
Comment (linda-seiter) [reply src/test/java/TVTest.java]: Yes, originally I was going to write tests with multiple tvs, but ended up going with just one that should be in the setup.
I'll add a test that does create a few tvs just to show why these need to be instance variables.

**Review by developerjaym — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -150,3 +168,19 @@ Run the tests to confirm your implementation:
...
+
+Remove `@Disabled` from the following Junit tests:
+
+- multipleTVsDifferentState
+
+Although the method passes if you run the test, you will need to implement
+the code for each comment to pass the tests when you submit this assignment.
+
+Add the appropriate method call and/or assertion below each comment.
+The method is testing two `Tv` objects referenced by variables `sony`
```
Comment (kcleland0818) [inline README.md]: `Tv` should be `TV` here

**Review by kcleland0818 — APPROVED** [review]
I like this last test! I think your instruction and direction are clear. The comments within the test really help and should guide the student along. 

Made note of just one small typo

File: README.md (hunk 2 above)
Comment (linda-seiter) [reply README.md]: thanks, fixed

**Review by AlveeM — APPROVED** [review]

**Pull request merged**